README.md

```diff
@@ -207,8 +207,11 @@ To run the tests against a remote deployment you will need to specify the enviro
 `EQ_FUNCTIONAL_TEST_ENV=https://staging-new-surveys.dev.eq.ons.digital/ yarn test_functional`
 
 ---
+## Deploying
 
-## Deployment with [Helm](https://helm.sh/)
+For deploying with Concourse see the [CI README](./ci/README.md).
+
+### Deployment with [Helm](https://helm.sh/)
```

Comment:
Most of the environment variables under the "Deploying the app" heading are no longer optional and are duplicated in the CI Readme. We should move the environment variables documentation that you've updated in the CI Readme to here and reference it from there. The CI specific variables `PROJECT_ID` and `REGION` should remain just in the CI Readme.